src/axom/mint/examples/user_guide/mint_getting_started.cpp

```diff
@@ -53,102 +53,102 @@ static struct
+mint::Mesh* getUnstructuredMesh();
 
 //------------------------------------------------------------------------------
 // PROGRAM MAIN
 //------------------------------------------------------------------------------
-int main ( int argc, char** argv )
+int main(int argc, char** argv)
 {
+  parse_args(argc, argv);
 
-  parse_args( argc, argv );
-
-// sphinx_tutorial_walkthrough_set_memory_start
+  // sphinx_tutorial_walkthrough_set_memory_start
   // NOTE: use unified memory if we are using CUDA
-  const int allocID = axom::execution_space< ExecPolicy >::allocatorID();
-  axom::setDefaultAllocator( allocID );
-// sphinx_tutorial_walkthrough_set_memory_end
+  const int allocID = axom::execution_space<ExecPolicy>::allocatorID();
+  axom::setDefaultAllocator(allocID);
+  // sphinx_tutorial_walkthrough_set_memory_end
 
-// sphinx_tutorial_walkthrough_construct_mesh_start
+  // sphinx_tutorial_walkthrough_construct_mesh_start
 
   mint::Mesh* mesh =
-    ( Arguments.useUnstructured ) ? getUnstructuredMesh( ) : getUniformMesh( );
+    (Arguments.useUnstructured) ? getUnstructuredMesh() : getUniformMesh();
 
-// sphinx_tutorial_walkthrough_construct_mesh_end
+  // sphinx_tutorial_walkthrough_construct_mesh_end
 
-// sphinx_tutorial_walkthrough_add_fields_start
+  // sphinx_tutorial_walkthrough_add_fields_start
 
   // add a cell-centered and a node-centered field
-  double* phi = mesh->createField< double >( "phi", mint::NODE_CENTERED );
-  double* hc  = mesh->createField< double >( "hc", mint::CELL_CENTERED );
+  double* phi = mesh->createField<double>("phi", mint::NODE_CENTERED);
+  double* hc = mesh->createField<double>("hc", mint::CELL_CENTERED);
 
   constexpr int NUM_COMPONENTS = 2;
-  double* xc  = mesh->createField< double >( "xc", mint::CELL_CENTERED, NUM_COMPONENTS );
+  double* xc =
+    mesh->createField<double>("xc", mint::CELL_CENTERED, NUM_COMPONENTS);
 
-// sphinx_tutorial_walkthrough_add_fields_end
+  // sphinx_tutorial_walkthrough_add_fields_end
 
-// sphinx_tutorial_walkthrough_compute_hf_start
+  // sphinx_tutorial_walkthrough_compute_hf_start
 
   // loop over the nodes and evaluate Himmelblaus Function
-  mint::for_all_nodes< ExecPolicy, xargs::xy >(
-      mesh, AXOM_LAMBDA( IndexType nodeIdx, double x, double y )
-  {
-    const double x_2 = x * x;
-    const double y_2 = y * y;
-    const double A   = x_2 + y   - 11.0;
-    const double B   = x   + y_2 - 7.0;
+  mint::for_all_nodes<ExecPolicy, xargs::xy>(
+    mesh,
+    AXOM_LAMBDA(IndexType nodeIdx, double x, double y) {
+      const double x_2 = x * x;
+      const double y_2 = y * y;
+      const double A = x_2 + y - 11.0;
+      const double B = x + y_2 - 7.0;
 
-    phi[ nodeIdx ] = A * A + B * B;
-  } );
+      phi[nodeIdx] = A * A + B * B;
+    });
 
-// sphinx_tutorial_walkthrough_compute_hf_end
+  // sphinx_tutorial_walkthrough_compute_hf_end
 
-// sphinx_tutorial_walkthrough_cell_centers_start
+  // sphinx_tutorial_walkthrough_cell_centers_start
 
```

Comment:
Example of formatting a more complicated lambda.

The body is untouched, and the heading seems cleaner to me.